src/classes/RD_AddDonationsBTN_TEST.cls

```diff
@@ -143,6 +144,85 @@ private with sharing class RD_AddDonationsBTN_TEST {
         Test.stopTest();
     }
 
+    /***
+    * @description Verifies if it displays error message when the uer has no create permission
```

Comment:
- Typo `uer`.
- Let's specify which SObject user needs the create permission for: ` * @description Verifies if it displays error message when the user has no create permission on the Opportunity SObject`
- Can you try to extract common code for all three methods and test and assert values? For example :
```
@isTest
    private static void shouldDisplayErrorMessageWhenUserHasNoCreatePermission() {
        UTIL_Permissions_TEST.Stub permissionStub = new UTIL_Permissions_TEST.Stub();
        permissionStub.hasCreate = false;

        testAndAssertUserDoesNotHavePermissions(permissionStub);
    }
```